_sources/Sequences/week2a1.rst

```diff
@@ -61,8 +61,9 @@ Chapter Assessment
     class myTests(TestCaseGui):
 
       def test_output(self):
-        self.assertEqual(last, ['curling', 'ping pong', 'hockey'], "Testing that the value of last is the last three items in sports.")
-        self.assertNotIn("[6:]", self.getEditorText(), "Testing that your code would work no matter how many items. (Don't worry about actual and expected values).")
+         import re
```

Comment:
Imports do not belong inside functions, except in rare circumstances.  Please move this outside.